.circleci/custom_scripts/build_service_image.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/env sh
+. ${CIRCLE_WORKING_DIRECTORY}/.circleci/custom_scripts/pull_base_image.sh
```

Comment:
i think it'd be more consistent with other scripts to put `common.sh` in this file. `pull_base_image.sh` could still be a separate file even though it's not use elsewhere

Reply:
Could do, but then it would be evaluated twice, once here and once in `pull_base_image.sh`

Reply:
i would have either removed it from pull_base_image.sh or actually got rid of pull_base_image.sh since it's not reused elsewhere. that way, you have one script per step. it's no big deal really, just a tidiness suggestion

Reply:
In my mind getting rid of it from `pull_base_image.sh` is a departure from consistency, and actually has a knock-on effect on functionality as then `pull_base_image.sh` can't be run by itself without erroring